docs/src/config/ini-config.adoc

```diff
@@ -423,33 +424,43 @@ must write RS274NGC code to standard output. This output is what will
 be displayed in the text area, previewed in the display area, and
 executed by LinuxCNC when Run.
 
-* 'PROGRAM_EXTENSION = .extension Description'
+----
+PROGRAM_EXTENSION = .extension Description
+----
 
 If your post processor outputs files in all caps you might want to add
 the following line:
 
-* 'PROGRAM_EXTENSION = .NGC XYZ Post Processor'
+----
+PROGRAM_EXTENSION = .NGC XYZ Post Processor
+----
 
 The following lines add support for the image-to-G-code converter
 included with LinuxCNC.
 
-* 'PROGRAM_EXTENSION = .png,.gif,.jpg Greyscale Depth Image'
-** 'png = image-to-gcode'
-** 'gif = image-to-gcode'
-** 'jpg = image-to-gcode'
+----
+PROGRAM_EXTENSION = .png,.gif,.jpg Greyscale Depth Image
+  png = image-to-gcode
+  gif = image-to-gcode
+  jpg = image-to-gcode
+----
```

Comment:
here it makes no sense having the descriptions as code, however, same as above comment

Reply:
Again, not descriptions but example of setting image-to-gcode as processor for the listed image formats.

Reply:
Okay I dug a bit into this and I revoke my statement. But if we change this to code, then we have to change this for all occurrences and that is a lot. That also would bloat the page gratuitously. So I still stick to not change any of them.

Only having the further lines as subitems is weird IMHO, I would suggest something like this:
```suggestion
* 'PROGRAM_EXTENSION = .png,.gif,.jpg Greyscale Depth Image'
+
'png = image-to-gcode'
+
'gif = image-to-gcode'
+
'jpg = image-to-gcode'
----
```